gensim/test/test_word2vec.py

```diff
@@ -50,6 +50,11 @@ def testfile():
     # temporary data will be stored to this file
     return os.path.join(tempfile.gettempdir(), 'gensim_word2vec.tst')
 
+def rule_for_testing(word, count, min_count):
+    if word == "human":
+        return 1  # throw out
```

Comment:
use the defs from word2vec module (`RULE_DEFAULT` etc); that's what they're for :)